test/onnx/expect/TestOperators.test_embedding_bags.expect

```diff
@@ -10,7 +10,6 @@ graph {
     output: "4"
     output: "5"
     output: "6"
-    name: "ATen_0"
```

Comment:
what happened here?..

Reply:
After the update for symbolics to remove ATen ops, test_embedding is only exported with ATEN_FALLBACK.
After this is changes, add_node_names is set to False, since we do not support add_node_names=True for operaotr export types other thatn ONNX.